oidc-client-app-samples/android-client-app-sample/gradle.properties

```diff
@@ -0,0 +1,13 @@
+# Project-wide Gradle settings.
```

Comment:
Missing license header.

Reply:
Fixed in PR: https://github.com/wso2/samples-is/pull/179